### ISSUE TITLE
Recognize `libblas.so.3` and `liblapack.so.3` 

### ISSUE THREAD

**Issue by pauljurczak** [alandefreitas/matplotplusplus#349]
I just installed Matplot++ from `Matplot++-1.2.0-Linux.sh` on Ubuntu 22.04.2, amd64 and gcc 12.1. `libblas.so` and `liblapack.so` are required just to include `<matplot/matplot.h>` header. Standard installation of these libraries provides only `libblas.so.3` and `liblapack.so.3` symbolic links, see similar issues: https://github.com/conda-forge/blas-feedstock/issues/29 and https://github.com/dealii/dealii/issues/3142. 

Could you recognize `libblas.so.3` and `liblapack.so.3`  or explicitly list `libblas-dev` and `liblapack-dev` as dependencies? This would help many users who are not aware of this library naming convention.

**Comment by alandefreitas** [comment]
Yes. The library will attempt to link to the libraries it was able to link when it generated the packages in CI. That includes optional dependencies found in CI as well as system libraries. 

The best solution here is to just build from source. 

In parallel, these packages should be clearer about which runner os they were built with or just be removed completely because I'm not sure they're making anything easier.
